doc/source/rllib/doc_code/getting_started.py

```diff
@@ -0,0 +1,111 @@
+# flake8: noqa
+
+# __rllib-first-config-begin__
+from ray.rllib.algorithms.ppo import PPOConfig
+from ray.tune.logger import pretty_print
+
+
+algo = (
+    PPOConfig()
+    .rollouts(num_rollout_workers=1)
+    .resources(num_gpus=0)
+    .environment(env="CartPole-v1")
+    .build()
+)
+
+for i in range(1000):
+    result = algo.train()
+    print(pretty_print(result))
+
+    if i % 100 == 0:
+        checkpoint_dir = algo.save()
+        print(f"Checkpoint saved in directory {checkpoint_dir}")
+# __rllib-first-config-end__
+
+
+# __rllib-tune-config-begin__
+import ray
+from ray import air, tune
+
+
+ray.init()
+
+config = {
```

Comment:
Shouldn't we also use an AlgorithmConfig object here to unconfuse the user?
e.g.
```
config = AlgorithmConfig().training(lr=tune.grid_search([0.01, 0.001, 0.0001]))
```
... should work just fine.

Reply:
The last thing we want to do is to tell the user they now have to additionally learn AlgorithmConfig, but also still master the old config dicts. These should all really disappear from our codebase and all our docs.

Reply:
for sure, totally agree. those are simply spots I forgot to clean up